tests/unit/lms/validation/authentication/_bearer_token_test.py

```diff
@@ -86,40 +89,19 @@ def test_it_raises_if_the_user_id_param_is_missing(self, schema, _jwt, param):
             "headers": {param: ["Missing data for required field."]},
         }
 
-    def test_serialize_and_deserialize_via_marshmallow_api(self, lti_user, schema):
```

Comment:
In this file I haven't done any effort to refactor the tests.

I could not resist to remove these tests you spotted where we seem to be testing something else. We seem to have quite a few of these.